### ISSUE TITLE
Adding a filter inside row api getTableRows

### ISSUE THREAD

**Issue by joanv1** [HubSpot/hubspot-api-nodejs#125]
Hi, 

I did some search and I found nothings, Is it possible to filter the function get Row, it has the comment which talk about that but nothings explain on where to put it, as example here is the function:
await hubspotClient.cms.hubdb.rowsApi.getTableRows(table, sort, after, limit, properties);

where do we put the filter here? I mean column__contain= for example.

Thanks for the help.

Kind regards.

**Comment by brtdwchtr** [comment]
@joanv1 did you manage to find an answer somewhere?

**Comment by joanv1** [comment]
> @joanv1 did you manage to find an answer somewhere?

Not at all. I filter manually but no way to do it on the function itself.

**Comment by KareyM** [comment]
I'm also having a weird issue where i'm getting this error after hitting this endpoint `await hubspotClient.cms.hubdb.rows.rowsApi.getTableRows(tableIdOrName, sort, after, limit, properties)` it gives me this error `Cannot read property 'rowsApi' of undefined`. Weirdly enough I don't get this error when I put this directly within the [documentation](https://developers.hubspot.com/docs/api/cms/hubdb) for testing, but when I run it locally, I get the error. If I switch from the node version to the url, it seems to work.

**Comment by thomas-j-moffett** [comment]
Running into this same issue as well. I went into the API code, and there doesn't seem to be anything that indicates these filters can be used. Very frustrating when filtering is clearly mentioned in the API documentation right under the **Retrieve HubDB Data** section, https://developers.hubspot.com/docs/api/cms/hubdb#retrieve-hubdb-data. Why not add this feature to the API??

**Comment by KareyM** [comment]
If anyone is having similar issue, the documentation here seem to be incorrect.  it should have been `hubspotClient.cms.hubdb.rowsApi`

**Comment by ajinvise** [comment]
A workaround until this is added:

```ts
const response = await hubspot.apiRequest({
    method: 'GET',
    path: `/hubdb/api/v2/tables/$tableId/rows`,
    qs: {
        name: "some-name",
    }
})

const results = await response.json()
console.log(results.objects)
```
